errors/template.go

```diff
@@ -68,6 +68,11 @@ func (t *Template) ServeError(errCode Code) http.Handler {
 		rw.Header().Set(HeaderErrorCode, fmt.Sprintf("%d - %q", errCode, errCode))
 
 		status := httpStatus(errCode)
+
+		if status == http.StatusUnauthorized {
```

Comment:
`WWW-Authenticate` header handling should find a place with basic auth context. The same http status code could occur for all other access control variants.

Reply:
Done